packages/php-db-import-export/src/Backend/Snowflake/SnowflakeException.php

```diff
@@ -38,7 +38,19 @@ public static function covertException(Throwable $e): Throwable
             );
         }
 
-        if (preg_match('/ \'(.*)\' cannot be inserted because it\'s bigger than column size/', $e->getMessage(), $output_array) === 1) {
+        $isOutOfRange = preg_match(
```

Comment:
Chtěl jsem napsat, že je to moc obecné, ale ono to reálně vybleje tu message uživateli, takže se tam snad v tom najde, kdyby to byl jiný problém

Reply:
Maximálně, jestli tam nechceš dát nakonec regexu `$`, že to tím má končit, ale to je možná blbost - třeba tam je pak ještě tečka nebo tak něco

Reply:
:D už sem to merdžnul :) 
Ale když to bude blbnůt pak v produkci tak to upravíme.